crates/ruff_linter/src/rules/ruff/rules/unused_noqa.rs

```diff
@@ -35,6 +35,9 @@ pub struct UnusedCodes {
 ///     foo.bar()
 /// ```
 ///
+/// ## Options
+/// - `external`
```

Comment:
Will this be hyperlinked automatically?

Reply:
Yeah I think it happens during our documentation generation

Reply:
Then this PR is...

https://media.giphy.com/media/fYlennPcKut8gjKOZD/giphy.gif

Reply:
Then this PR is...

![approved gif](https://media.giphy.com/media/fYlennPcKut8gjKOZD/giphy.gif)